packages/convenient_test_manager/lib/misc/setup.dart

```diff
@@ -34,8 +30,6 @@ Future<void> setup({
 
   if (initVLC) await DartVLC.initialize();
 
-  if (setWinSize) await _setWindowSize();
```

Comment:
IMHO it is great to set to the bigger windows size (e.g. the original code - 1350x1000) on larger screens, because the default window size is a bit too small to be used on larger screens. What do you think? One way is to detect screen resolution, and set windows size according to that

Reply:
> The problem with your (the existing) solution is, that the window then cannot be resized below that size because flutter thinks that that is the minimum size.

Yes, we should allow users to resizes arbitrarily. I meant that, maybe we can provide a bigger *initial* size.